app/assets/javascripts/rad_common/dynamic_updater.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready( function() {
+    $('form.dynamic-updater input').change(function() {
+        let url = $(this).closest('form').attr('action') + ".json";
+        let data = $(this).closest('form').serializeArray();
+        $.post(url, data)
+            .fail(function(data) {
+                addAlert('danger', data.responseJSON.error)
+            });
+    });
+
+    function addAlert(type, message) {
```

Comment:
This is kinda funky, the flash messages stay on the page. It's a rare case to get an error, let's just use an alert box and also need to undo the checkbox selection if it fails.

Reply:
done